### ISSUE TITLE
How to make no error in code mode

### ISSUE THREAD

**Issue by djkloop** [josdejong/jsoneditor#541]
![image](https://user-images.githubusercontent.com/21050914/41584524-0a6f5058-73da-11e8-9ffd-a85957a51b10.png)

**Comment by djkloop** [comment]
@josdejong thx.
I caught the exception with the Trycatch and solved the problem.
But I still have a problem with the ACE editor I use but he has no intelligent error, how can I set it?
![image](https://user-images.githubusercontent.com/21050914/41646960-a6c36fca-74a7-11e8-9080-f665b2db9cc6.png)

This is the online editor provided by the official website
![image](https://user-images.githubusercontent.com/21050914/41647074-f488ebd6-74a7-11e8-9d5f-3fb2d19471d1.png)

I made the JSON editor inside the Vue, and my settings are as follows
![image](https://user-images.githubusercontent.com/21050914/41647128-1884325c-74a8-11e8-8c6b-c787295cfdfe.png)
